plugins/org.wso2.developerstudio.humantaskeditor/HumanTaskEditor/js/WSDLOperations.js

```diff
@@ -101,10 +103,13 @@ function generateInputWSDL(wsdlDom, currentTask) {
     serviceName = currentTask+"Service";
     portName = currentTask + "Port";
     inputElements = $('#' + currentTaskName + ' #inputmappingTable tr');
-
+    targetnamespace = xmlDom.getElementsByTagName("humanInteractions")[0].getAttribute("targetNamespace");
+    wsdlDom.getElementsByTagName("definitions")[0].setAttribute("xmlns:tns",targetnamespace);
+    wsdlDom.getElementsByTagName("definitions")[0].setAttribute("targetNamespace",targetnamespace);
```

Comment:
Thanks. Array length checks were added in https://github.com/wso2/devstudio-tooling-bps/pull/30/